cigaradvisor/blocks/authorcard/authorcard.css

```diff
@@ -0,0 +1,224 @@
+.authorcard-container {
+    background: url('/cigaradvisor/images/author-card/media_17501a5f7bf5b4aebcf109765ead7cb5956480cbe.png');
+    margin-top: 50px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-color: transparent;
+    padding: 0 40px;
+    background-size: 900px 500px;
+    margin-bottom: 40px;
+}
+
+.authorcard-wrapper::after {
+    display: block;
+    clear: both;
+}
+
+.authorcard-wrapper {
+    display: flex;
+    padding-left: 0;
+    padding-right: 0;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 1080px;
+    align-items: stretch;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    align-content: flex-start;
+}
+
+.authorcard.block {
+    display: flex;
+}
+
+.authorcard.block .left-column,
+.right-column {
+    display: flex;
+    float: none;
+    width: 50%;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 10px;
+}
+
+.authorcard.block .left-column h2 {
+    font-family: "alfa slab one", cursive;
+    font-size: 40px !important;
```

Comment:
We shouldn't need to use `!important` anywhere. 

We should evaluate what makes sense for consistency of font sizes for h1/h2/etc

Then we can have variables for header font sizes for s, s, m, l, xl - depending on how many and which ones we need.

Reply:
fixed.